src/ext/java/org/opentripplanner/ext/vectortiles/layers/vehiclerental/VehicleRentalLayerBuilder.java

```diff
@@ -39,7 +39,7 @@ public VehicleRentalLayerBuilder(Graph graph, VectorTilesResource.LayerParameter
   protected List<Geometry> getGeometries(Envelope query) {
     VehicleRentalStationService service = graph.getVehicleRentalStationService();
     if (service == null) {return List.of();}
-    return service.getVehicleRentalStations()
+    return service.getVehicleRentalPlaces()
```

Comment:
We should probably split this layer in two, so that they can be showed on different zoom levels. This can be done later